medusa/helpers/__init__.py

```diff
@@ -662,7 +662,7 @@ def get_absolute_number_from_season_and_episode(show, season, episode):
     :param episode: Episode number
     :return: The absolute number
     """
-    from . import db
+    from medusa import db
```

Comment:
These are to avoid circular imports due to bad package design.  It will be addressed in the future when this package gets split into individual components